train.py

```diff
@@ -59,18 +41,27 @@ def main(window_size, batch_size, ep_count, model_name, pretrained, debug):
 
 
 if __name__ == "__main__":
-    args = docopt(__doc__)
-
-    months = args["<months>"]
-    window_size = int(args["--window-size"])
-    batch_size = int(args["--batch-size"])
-    ep_count = int(args["--episode-count"])
-    model_name = args["--model-name"]
-    pretrained = args["--pretrained"]
-    debug = args["--debug"]
-    stock_name = args["--stock-name"]
-
-    api = tradeapi.REST()
```

Comment:
These changes are unrelated to the runfile

Reply:
Ok, I have fixed these. Although, they are related to the run file as the commands were giving an error, and if i add them back, then I will need to update the run file as well.

Reply:
I'll try and come up with a fix for this, it's just not a viable solution as of now.